controllers/operatorconfig_controller.go

```diff
@@ -18,15 +18,37 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"time"
+
 	"github.com/go-logr/logr"
+	cranev1alpha1 "github.com/konveyor/mtrho-operator/api/v1alpha1"
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
+	pipelinev1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+)
 
-	cranev1alpha1 "github.com/konveyor/mtrho-operator/api/v1alpha1"
+const (
+	Finalizer         = "openshift.konveyor.crane"
+	OwnerConfigName   = "openshift-migration"
+	InvalidConfigName = "InvalidConfigName"
+	True              = "True"
```

Comment:
nit: this is not needed as we have https://pkg.go.dev/k8s.io/api/core/v1#ConditionStatus the constant available already, might want to use those